packages/test/src/workflows/log-sink-tester.ts

```diff
@@ -34,3 +34,10 @@ export async function logSinkTester(): Promise<void> {
     }`
   );
 }
+
+// eslint-disable-next-line deprecation/deprecation
+const { defaultWorkerLogger } = wf.proxySinks<wf.LoggerSinks>();
+
+export async function useDepreatedLoggerSinkWorkflow(): Promise<void> {
+  defaultWorkerLogger.info(`Log message from workflow`, { workflowId: wf.workflowInfo().workflowId });
```

Comment:
nit:
```suggestion
  defaultWorkerLogger.info('Log message from workflow', { workflowId: wf.workflowInfo().workflowId });
```

Reply:
We should have a lint rule that prevents this for consistency (not sure if it affects performance even if it's irrelevant in this test).

Reply:
>We should have a lint rule that prevents this for consistency (not sure if it affects performance even if it's irrelevant in this test).

FYI it has absolutely no performance impact. In V8, template strings get segmented at parse time. If the template string contains no expression segment, the parser emit a `StringLiteral` AST node rather than a `TemplateLiteral` node.

But yes, I agree with the consistency concern. Will add a eslint rule for that in a different PR, to avoid polluting this one with tons of insignificant quote changes.